test/spec/factory/EntryFieldDescriptionSpec.js

```diff
@@ -129,6 +129,62 @@ describe('factory/EntryFieldDescription', function() {
     });
 
 
+    it('should preserve line breaks', function() {
+
+      // when
+      var html = EntryFieldDescription('Hello <br/> world');
+
+      // then
+      expect(html).to.eql(
+        '<div class="bpp-field-description">' +
+          'Hello <br /> world' +
+        '</div>'
+      );
+    });
+
+
+    it('should ignore whitespaces in br tag', function() {
+
+      // when
+      var html = EntryFieldDescription('Hello <br       /> world');
+
+      // then
+      expect(html).to.eql(
+        '<div class="bpp-field-description">' +
+          'Hello <br /> world' +
+        '</div>'
+      );
+    });
+
+
+    it('should preserve line breaks within <a> text', function() {
+
+      // when
+      var html = EntryFieldDescription('<a href="https://test.com"> HELLO <br/> WORLD </a>');
+
+      // then
+      expect(html).to.eql(
+        '<div class="bpp-field-description">' +
+          '<a href="https://test.com" target="_blank"> HELLO <br /> WORLD </a>' +
+        '</div>'
+      );
+    });
+
+
+    it('should ignore <a> if br used within URL field', function() {
+
+      // when
+      var html = EntryFieldDescription('<a href="https://test<br />website.com"> HELLO <br/> WORLD </a>');
```

Comment:
Is this what we want? I'd rather URL-escape the BR tag in that case.

Reply:
Here the thing is that our URL detector pattern does not accept URL's with **<** or **>** in it due to:
`<a href="(https?:\/\/[^"<>]+)">`

So we actually handle those as plain text, so the <br /> just adds a new line. 

Independent from the BR tag, this also does not work:
`<a href="https://www.<.com>"> SDFDSF </a>`

Reply:
O.k. can we make it work then?

Reply:
So If I got your point correctly what we want is that:

<a href="https://test<br />site.com"> TEST </a>

Should be considered as a valid URL, but when we click on TEST, we'd go to `test<br />site.com`?

Reply:
Yes, this is what I'd consider predictable behavior. Is it hard to implement? If so, I'm happy if we stick to the current behavior for now.

Reply:
I integrated this behavior into <a> tags but did not change the current behavior for markdown as the existing `< or >` characters causes problems while being converted into <a> element and there was an existing test making sure that markdowns with HTML chars should be ignored: 

> should ignore special HTML chars in markdown link